JitsiConference.js

```diff
@@ -142,10 +148,11 @@ function JitsiConference(options) {
 
     /**
      * Flag set to <tt>true</tt> when P2P session has been established
-     * (ICE has been connected).
+     * (ICE has been connected) and this conference is currently in the peer to
+     * peer mode (P2P connection is the active one).
      * @type {boolean}
      */
-    this.p2pEstablished = false;
+    this.p2p = false;
```

Comment:
Is the description of this field wrong ? For me it looks good, but if you have suggestions please give something specific how it can be improved.